docs/modules/ROOT/pages/references.adoc

```diff
@@ -1,98 +1,115 @@
+. [[ref:63]] Thijs, G. (2019) OSLO MODELLERINGSREGELS, sec 3.1.11. Available at: https://github.com/Informatievlaanderen/OSLO-handleiding/blob/master/Modellering/OSLO-Modelleringsregels.pdf.
+. [[ref:64]] Buitelaar.P et al. (2011). Ontology Lexicalisation: The lemon Perspective.
 . [[ref:65]] Vann
-. [[ref:66]] https://www.w3.org/TR/shacl/
-. [[ref:67]] https://shex.io/
+. [[ref:66]] Knublauch, H. and Kontokostas, D. (eds) (2017) Shapes constraint language (SHACL), W3C. W3C. Available at: https://www.w3.org/TR/shacl/.
+. [[ref:67]]Shape expressions(ShEx). Available at: https://shex.io/ (Accessed: January 2, 2023).
 . [[ref:68]] R. Reiter. On closed world data bases. In Readings in artificial intelligence, pages 119–140. Elsevier, 1981.
-. [[ref:69]] TOGAF
-. [[ref:70]] Methontology
-. [[ref:71]] DOGMA
-. [[ref:72]] OntoClean
+. [[ref:69]] TOGAF. (2022) The Open Group Website. Available at: https://www.opengroup.org/togaf (Accessed: January 2, 2023).
+. [[ref:70]] López, F et al. (1997). METHONTOLOGY: from ontological art towards ontological engineering. Engineering Workshop on Ontological Engineering (AAAI97).
+. [[ref:71]] Jarrar, M et al. (2008). Ontology engineering - The DOGMA approach. 10.1007/978-3-540-89784-2_2.
+. [[ref:72]] Guarino, N et al. (2009). An Overview of OntoClean. 10.1007/978-3-540-92673-3_9.
 . [[ref:73]] TOVE - Fox, M.S., (1992), “The TOVE Project: A Common-sense Model of the Enterprise”, Industrial and Engineering
 Applications of Artificial Intelligence and Expert Systems, Belli, F. and Radermacher, F.J. (Eds.), Lecture Notes in
 Artificial Intelligence # 604, Berlin: Springer-Verlag, pp. 25-34.
-. [[ref:74]] DILIGENT
-. [[ref:75]] NeOn
-. [[ref:76]] https://www.researchgate.net/publication/226919523_Ontology_Engineering_Methodology
-. [[ref:77]] DAMA-DMBOK
-. [[ref:78]] Zachman Framework
-. [[ref:79]] EAP
-. [[ref:80]] OSLO spec, sec 3.1.4
-. [[ref:81]] OSLO spec, sec 3.1.3
-. [[ref:82]] ePO docs
-. [[ref:83]] https://5stardata.info/en/
-. [[ref:84]] Draft styleguide 8.2
-. [[ref:85]] https://www.w3.org/TR/dwbp/
-. [[ref:86]] ref EU PURI
-. [[ref:87]] https://www.w3.org/TR/dwbp/#DataIdentifiers
+. [[ref:74]] Pinto, H. et al. (2004). DILIGENT: Towards a fine-grained methodology for Distributed, Loosely-controlled and evolving Engineering of oNTologies.. Proceedings of the 16th European Conference on Artificial Intelligence (ECAI 2004). 393-397.
+. [[ref:75]] Baonza, M. (2010). NeOn Methodology for Building Ontology Networks:
+Specification, Scheduling and Reuse.
+. [[ref:76]] Vetter, S et al., Rudi. (2009). Ontology Engineering Methodology. 10.1007/978-3-540-92673-3_6..
+. [[ref:77]] Dama-dmbok: Data Management Body of Knowledge (2017). New Jersey: Technics Publications.
+. [[ref:78]]  Zachman, J.A. "A Framework for Information Systems Architecture." IBM Systems Journal, Volume 26, Number 3.
```

Comment:
Decide on the usage of quote signs (") for the publication titles.

Reply:
Better to choose a citation format and stick to it. Also, decide on a minimum set of fields and stick to them.